### ISSUE TITLE
Is it bad to change batch size, width and height in mid training?

### ISSUE THREAD

**Issue by compnaion** [pjreddie/darknet#1842]
I was having ave.loss nan. When I started training I changed the width and height to 608, (somewhere it said, it will make the training better.). But after I got nan, I started the training with width and height equal to 416 and increased the batch size. Is it a bad thing to do? I still cannot get a good prediction, if at all, btw.

**Comment by NABDroid** [comment]
Hi, did you found any solution? I want to change the width and height. Will it be any problem?

**Comment by compnaion** [comment]
> Hi, did you found any solution? I want to change the width and height. Will it be any problem?

It's been a long time and I stopped using this and went for tensorflow and Faster R-CNN. But my knowledge on object has somewhat increased too. If I had to guess, I would say no, it doesn't affect the training. After all, algorithm extracts features from the image. Feature maps for objects may vary somewhat, but end result would probably depend more on your dataset than configuration. If the detection rate is low, increase the number of examples.
